src/pykeen/lr_schedulers/__init__.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+"""Learning Rate Schedulers available in PyKEEN."""
+
+from typing import Any, Mapping, Type
+
+from class_resolver import Resolver
+from torch.optim.lr_scheduler import (
+    CosineAnnealingLR, CosineAnnealingWarmRestarts, CyclicLR, ExponentialLR, LambdaLR, MultiStepLR, MultiplicativeLR,
+    OneCycleLR, StepLR, _LRScheduler,
+)
+
+__all__ = [
+    'LRScheduler',
+    'lr_schedulers_hpo_defaults',
+    'lr_scheduler_resolver',
+]
+__all__.extend((
+    subcls.__name__
+    for subcls in _LRScheduler.__subclasses__()
+    if subcls.__name__ != 'SWALR'
+))
+
+#: A wrapper around the hidden scheduler base class
+LRScheduler = _LRScheduler
+
+#: The default strategy for optimizing the lr_schedulers' hyper-parameters
+lr_schedulers_hpo_defaults: Mapping[Type[_LRScheduler], Mapping[str, Any]] = {
```

Comment:
```suggestion
lr_schedulers_hpo_defaults: Mapping[Type[LRScheduler], Mapping[str, Any]] = {
```
or is there a problem with this?

Reply:
https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases